lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_download_file_by_version.rb

```diff
@@ -57,6 +60,12 @@ def self.available_options
                                        description: 'The prefix which is used in the GitHub release title',
                                        type: String,
                                        optional: true),
+          FastlaneCore::ConfigItem.new(key: :github_token,
+                                       env_name: 'GITHUB_TOKEN',
+                                       description: 'The GitHub OAuth access token',
+                                       optional: false,
+                                       default_value: false,
+                                       type: String),
```

Comment:
Self-Review: As stated on the P2, I'm still not certain if I should use at the env_name var the `GITHUB_TOKEN` or `GHHELPER_ACCESS`

Reply:
As answers on the internal P2 post, I see two possible options:
 - Use `env_names` instead of `env_name` (which I learned about just today while looking at [fastlane source code](https://github.com/fastlane/fastlane/blob/master/fastlane_core/lib/fastlane_core/configuration/config_item.rb) 😅 ) to allow still supporting both env vars and avoid breaking changes
 - Use the occasion to finally get rid of the old and deprecated `GHHELPER_ACCESS` env var, even if it means breaking changes (we follow [SemVer](https://semver.org/) on our `release-toolkit` lib, so that's ok)

Reply:
Yeah, As I said there, as we are all comfortable with the change to github_token, I'll change it and we can always go back if we see any side effects 😄